dolly-backend-app/src/main/java/no/nav/dolly/bestilling/inntektstub/InntektstubClient.java

```diff
@@ -25,7 +24,6 @@
 import static org.apache.commons.lang3.StringUtils.isNotBlank;
 
 @Slf4j
-@Order(4)
```

Comment:
Måtte ikke denne være med for at dagpengevedtak skulle gå gjennom? 😅

Reply:
Opps, det stemmer sikkert, legger den tilbake. 💸